x-pack/plugins/cases/public/components/configure_cases/index.tsx

```diff
@@ -156,19 +156,21 @@ const ConfigureCasesComponent: React.FC<ConfigureCasesProps> = ({ userCanCrud })
     }
   }, [connectors, connector, isLoadingConnectors]);
 
-  const ConnectorAddFlyout = useMemo(
+  const ConnectorAddFlyout = useCallback(
```

Comment:
Why the change from `useMemo` to `useCallback`?

Reply:
we tried something and it need to come back...